### DISCUSSION TITLE
Historic Data Inventory

### DISCUSSION THREAD

**Discussion by LauraNWRM** [NewGraphEnvironment/restoration_wedzin_kwa_2024#39] [General]
Hi all,
You can find the data inventory [here](https://docs.google.com/spreadsheets/d/15kXMqZzAsL1cnbute1rP0qwfMkZTWqnxgo7gD0FGFUg/edit?usp=sharing)
Two questions: 

Now that I've inventoried all the MWMT data, including that which is not posted online, I could use some direction in what needs to added. Since Al is pulling all relevant data from SKT directly, do we need to be searching and adding that publicly available data to the inventory list as well? 

Has data from PSF been pulled?

**Comment by NewGraphEnvironment** [comment]
Great google sheet. Thank you!!

I think making a column on the google sheet that flags any data that is not on SKT would be helpful.  Particularly if you are interested in either a. loading that data to SKT eventually and/or b. loading data not on SKT to somewhere else (ex. [aws s3](https://aws.amazon.com/s3/?gclid=CjwKCAjw17qvBhBrEiwA1rU9wxeo-iwc2qdfn6bjZeHHS38t_xJGQwDRa5np0Dl3RWmVluRrn6aOABoCVt8QAvD_BwE&trk=e747cc26-a307-4ae0-981a-6dc5c1cb4121&sc_channel=ps&ef_id=CjwKCAjw17qvBhBrEiwA1rU9wxeo-iwc2qdfn6bjZeHHS38t_xJGQwDRa5np0Dl3RWmVluRrn6aOABoCVt8QAvD_BwE:G:s&s_kwcid=AL!4422!3!453053794209!e!!g!!aws!10705896207!102406402981)  or [digital ocean spaces](https://try.digitalocean.com/cloud-storage/?utm_campaign=amer_brand_kw_en_cpc&utm_adgroup=digitalocean_spaces_phrase&_keyword=digital%20ocean%20cloud%20storage&_device=c&_adposition=&utm_content=conversion&utm_medium=cpc&utm_source=google&gad_source=1&gclid=CjwKCAjw17qvBhBrEiwA1rU9wyb-s7raGKq_IoUF2oiqo6Eo-qEiIXcc0j-w4WD6eB-M67W9UqQ4VxoCy7AQAvD_BwE))


For PSF - are you referring to data from their [document library](https://psf.ca/document-library/) and the [`Pacific Salmon Explorer`](https://www.salmonexplorer.ca/#!/)?  If there are particular resources we might be interested in there - those could perhaps be referenced in the google sheet by PSF url (if they are not already on SKT - if they are we can use the SKT url) and then if we want downloaded resources we grab them directly from PSF or SKT when possible?  

If you are going to add more resources to the `data inventory google sheet` and the `Synology - gofile` system  - perhaps adding a `date_uploaded` column and using a date format like `20240311` (yyyymmdd) would be helpful for letting us know what is newly added.... 

Regarding the historic restoration site data we have downloaded to `onedrive` from the `Synology - gofile` system - Looks like perhaps an issue with the 2021 monitoring data... `Monitoring.zip` doesn't seem to want to `unzip`. @LauraNWRM - could you have a look and see if you have the same issue on your end and if so - if you might be able to sub in the data to the same place on `Synology - gofile` that we can re-download and unzip?

![image](https://github.com/NewGraphEnvironment/restoration_wedzin_kwa_2024/assets/46538103/13040f0c-50a9-45a3-a922-1a17b362bd1e)

**Reply by LauraNWRM** [reply]
The 'Monitoring' file has since been unzipped and subfolders rezipped since you last downloaded it
